src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -660,12 +684,18 @@ const StudentApplication = ({ userProfile, authentication }) => {
                   </Grid>
                 </Grid>
                 <Grid item xs={12} md={4}>
-                  <Card>
-                    <CardHeader title="Agreements" className="card-header" />
-                    <CardContent>
-                      <Typography variant="body1">Placeholder text</Typography>
-                    </CardContent>
-                  </Card>
+                  <Collapse
```

Comment:
Why did you change the agreements to be under this Collapse header?

Reply:
Oh, I think this is because it does not need to show the agreements if the current user is not the editor (And checking for whether the user is the editor is a part of the "load application" feature

Reply:
This comment/review should go in #963